bindings/nodejs/lib/types/secretManager/secretManager.ts

```diff
@@ -31,3 +33,14 @@ export type SecretManagerType =
     | MnemonicSecretManager
     | StrongholdSecretManager
     | PlaceholderSecretManager;
+
+export interface EvmSignature {
+    /**
+     * The public key.
+     */
+    publicKey: HexEncodedString;
+    /**
+     * The signature.
+     */
+    signature: HexEncodedString;
```

Comment:
Can we get this aligned with ledger, so that it respectively returns the `v`, `r`, `s`  values? What's the reason for including the publicKey?

Reply:
This is aligned with Ed25519 signatures in the protocol, they also contain the public key so you can get back the address associated with it